src/content/api/businesses.mdx

```diff
@@ -34,11 +34,11 @@ A Business represents a company registered with the New Zealand Companies Office
   </Property>
 
   <Property name="name" type="string">
-    Legal name recorded in the Companies Register.
+    The legal name of the business.
```

Comment:
For sole trader I'm not sure what this would be I think it might be an individual name

Reply:
Does a sole trader not have a legal name?

Reply:
Nah I don't think so they pay their business taxes under their personal income tax